eppy/tests/test_bunch_subclass.py

```diff
@@ -379,6 +380,21 @@
     CLEAR 3MM,               !- Outside Layer
     AIR 13MM,                !- Layer 2
     CLEAR 3MM;               !- Layer 3
+    
+Zone,
```

Comment:
For testing, I added a zone ep object to the `idftxt` variable

Reply:
You could have a fixture building off this, so you'd then have a fixture which has a zone object, without affecting the tests that don't need it. Although there's actually quite a bit of test cleaning/simplifying we should do around using fixtures, so this is probably fine for now.